pipelines/botbuilder-python-ci-slack-test.yml

```diff
@@ -0,0 +1,104 @@
+#
+# Runs functional tests against the Slack channel.
+#
+
+# "name" here defines the build number format. Build number is accessed via $(Build.BuildNumber)
+name: $(Build.BuildId)
+
+pool:
+  vmImage: $[ coalesce( variables['VMImage'], 'windows-2019' ) ] # or 'windows-latest' or 'vs2017-win2016'
+
+trigger: # ci trigger
+  batch: true
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - '*'
+    exclude:
+    - doc/
+    - specs/
+    - LICENSE
+    - README.md
+    - UsingTestPyPI.md
+
+pr: # pr trigger
+  branches:
+    include:
+    - main
+  paths:
+    include:
+    - pipelines/botbuilder-python-ci-slack-test.yml
+
+variables:
+  AppId: $(SlackTestBotAppId)
+  AppSecret: $(SlackTestBotAppSecret)
+  BotGroup: $(SlackTestBotBotGroup)
+  BotName: $(SlackTestBotBotName)
+  SlackBotToken: $(SlackTestBotSlackBotToken)
+  SlackClientSigningSecret: $(SlackTestBotSlackClientSigningSecret)
+  SlackVerificationToken: $(SlackTestBotSlackVerificationToken)
```

Comment:
Do we need these 3 here? I think that these are being used like that only for the script in line 87 and these variables are declared there

Reply:
They are used in the resource creation step, ported from the DotNet implementation.
They are indeed redundant, we could replace them.